onmetal-controller-manager/controllers/networking/loadbalancer_controller.go

```diff
@@ -85,8 +86,15 @@ func (r *LoadBalancerReconciler) reconcile(ctx context.Context, log logr.Logger,
 	}
 	log.V(1).Info("Successfully found destinations", "Destinations", destinations)
 
+	log.V(1).Info("Finding network")
+	network, err := r.findNetwork(ctx, log, loadBalancer)
+	if err != nil {
+		return ctrl.Result{}, fmt.Errorf("error finding network: %w", err)
```

Comment:
We might want to print out which network we wanted to find.

Reply:
done